### ISSUE TITLE
Define python versions supported

### ISSUE THREAD

**Issue by kikocorreoso** [monocongo/climate_indices#66]
It is not pretty clear which python versions are supported right now.

As it is a new package I think you should support most moder python versions, Python >= 3.5, but it should be clariffied somewhere (docs, readme, trove classifiers in setup.py,...)

I've added python 3.5 and 3.6 in #64 but it hould be ammended iy you have other plans.

**Comment by monocongo** [comment]
Yes, agreed, it's mentioned in the README that this is Python 3 but we can be more explicit. We can include a badge of some sort on the README showing the versions supported.
